### ISSUE TITLE
ICp successfully deployed in local VMs using Vagrant on Mac.

### ISSUE THREAD

**Issue by DouglasGulledge** [IBM/deploy-ibm-cloud-private#146]
ICp successfully deployed in local VMs using Vagrant on Mac.
I had to modify the Vagrantfile parameter base_segment from 192.168.27 to 192.168.56.
This is the ONLY change I made.
From a MacOS terminal, I can't ping 192.168.56.100; so obviously can't access the ICp admin console.
I am looking for some guidance.

**Comment by DouglasGulledge** [comment]
I have attempted to replicate the success as documented in #147 

#sigh  This didn't resolve my issue.
https://192.168.16.100:8443/console/dashboard isn't navigable.

|  NAME   |  STATE  |         IPV4          | IPV6 |    TYPE    | SNAPSHOTS |
+---------+---------+-----------------------+------+------------+-----------+
| worker1 | RUNNING | 192.168.16.101 (eth0) |      | PERSISTENT | 0         |
|         |         | 172.17.0.1 (docker0)  |      |            |           |
|         |         | 10.1.235.128 (tunl0)  |      |            |           |
+---------+---------+-----------------------+------+------------+-----------+
| worker2 | RUNNING | 192.168.16.102 (eth0) |      | PERSISTENT | 0         |
|         |         | 172.17.0.1 (docker0)  |      |            |           |
|         |         | 10.1.189.64 (tunl0)   |      |            |           |

Please advise.

**Comment by DouglasGulledge** [comment]
I now have a working configuration, here are my observations, your mileage may vary.

- I didn't need to make ANY changes to a host file.  I did review a couple of Issues raised, but seemingly resolved, with pertinent changes to the Vagrantfile available in the latest code base.

- I leveraged the Vagrantfile "as is", with ONLY these three changes:
a) license = 'accept',  b) cpus = '6'  (was '4'),  c) memory = '12288' (was '10240'),
d) base_segment = '192.168.56' (was '192.168.27').

- (This seemed significant for me).  I rebooted the Mac after installing Virtualbox.  In debugging with a peer, this seemed necessary as for some reason the kernel changes weren't taking affect (until the reboot).  This was manifest in an inability for Virtualbox to update the MacOs network routing table.

- (MOST Important)  MacOS High Sierra now includes a Firewall.  System Pref > Security & Privacy.  I needed to make these two changes to the Firewall Policies:
a) Add Virtualbox so as to "Allow Incoming connections"
b) I disabled Stealth Mode to support test applications using ICMP, such as Ping.
